app/src/main/java/com/compscieddy/meetinthemiddle/AuthenticationActivity.java

```diff
@@ -17,10 +24,11 @@
 /**
  * Created by ambar on 6/15/16.
  */
-public class AuthenticationActivity extends Activity {
+public class AuthenticationActivity extends AppCompatActivity implements DialogInterface.OnDismissListener {
 
   private static final Lawg lawg = Lawg.newInstance(AuthenticationActivity.class.getSimpleName());
   private static final int RC_SIGN_IN = 100;
+  private static final String TAG_INTERNET_ERROR = "fragment_internet_error";
```

Comment:
I think it's good practice to make the string the same as the constant name - prevents any accidental duplicate naming of the TAGs (which may not be an issue now, but could be later)